src/main.rs

```diff
@@ -24,7 +24,7 @@ fn main() {
         addstr("---------------\n");
         addstr("a: Add, d: Delete, x: Done/Undone, j: DOWN, k: UP, q: Quit\n\n");
 
-        if todos.len() == 0 {
```

Comment:
Thanks for the pull request. As I am new to Rust, will you explain the difference between this and is_empty?

Reply:
It was a suggestion provided by running `cargo clippy` ([crate](https://github.com/rust-lang/rust-clippy)).

`foo.is_empty()` is essentially returning the result of `foo.len() == 0`. The clippy suggestion motivated the change by saying something like "is_empty makes the code more readable and self-documenting".

Given that you're new to Rust, here is a workflow suggestion I wish I could give myself when I was first starting out. 

1. Checkout to where this PR started from
2. Fix the compile error (trivial)
3. Run `cargo clippy` (and look into more pedantic 
4. Critically evaluate each suggestion/warning
5. Treat each like a TODO item - apply suggestion or some alternative to address it, such as:
 * Annotating with `#[allow(clippy::$foo)]`
 * Improving implementation strategy.
 * Commenting code explaining why you want to keep the warning short term
6. run `cargo fmt` (applies `rustfmt` to your project)
7. automate your workflow using [cargo-make](https://sagiegurari.github.io/cargo-make/)
 * I first saw it put to excellent use in the [seed](https://github.com/seed-rs/seed/blob/master/Makefile.toml) library.

I haven't had `clippy` in my workflow for a very long time, so I don't know how to get the most out of it. With that said, when I first started using `clippy` it felt very similar to when I first started using rust with the warnings that gave out - it taught me a lot about the how and why of _writing better code_. 

As for `fmt`, it keeps things standardized, which is something I _love_. I don't see style-~wars~debates in Rust like I sometimes see in C. Any discussion comes with the implicit statement "default to standard `rustfmt` for your project, and configure to taste`."